include/lsst/ip/diffim/DipoleAlgorithms.h

```diff
@@ -208,7 +208,9 @@ inline DipoleFluxAlgorithm::DipoleFluxAlgorithm(
 class that knows how to calculate centroids as a simple unweighted first
  * moment of the 3x3 region around the peaks
  */
-class NaiveDipoleFlux : public DipoleFluxAlgorithm {
+class [[deprecated(
+        "This algorithm is deprecated and will be removed after v27.")]]
```

Comment:
Could you include the names of the algorithms in the deprecation warnings?

Reply:
That comes "for free" with the compiler warning that is emitted when you try to use the class.